.travis.yml

```diff
@@ -17,62 +17,21 @@ jobs:
         - DESC="acceptance tests"
         - CMD="./mvnw install --projects acceptance-tests --also-make --activate-profiles all --no-transfer-progress"
 
-    - jdk: oraclejdk8
-      env:
-        - DESC="findbugs"
-        - CMD="./mvnw install findbugs:check --projects '!scala-unit-tests,!jmh-scala-tests,!jmh-tests,!jcstress-tests,!p2-repository' --activate-profiles all -DskipTests=true --no-transfer-progress"
-
-    - jdk: oraclejdk8
-      env:
-        - DESC="checkstyle"
-        - CMD="./mvnw install checkstyle:check --activate-profiles all -DskipTests=true --no-transfer-progress"
-
-    - jdk: oraclejdk8
```

Comment:
Please leave the unit tests for Oracle JDK 8

Reply:
Done